Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 categories = ["cryptography::cryptocurrencies", "development-tools::testing"]
 
 [dependencies]
-bitcoincore-rpc = { version = "0.18.0" }
+bitcoincore-rpc = { version = "0.19.0" }
```

Comment:
Since there are no source code changes I wonder if we can use a ranged version here.

Reply:
Interesting thought, I'll give it a shot.

Reply:
It works back to 0.17.0 (see https://github.com/rust-bitcoin/bitcoind/pull/158).